src/components/App/BrowserNotification/NotificationEnablingPage.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="notification-enabling-pointer">
+    <div class="notification-enabling-pointer__instructions">
+      <div class="notification-enabling-pointer__text">{{ t('browserNotificationDialog.pointer') }}</div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+
+import TranslationMixin from '@/components/mixins/TranslationMixin';
+
+@Component
+export default class NotificationEnablingPage extends Mixins(TranslationMixin) {}
+</script>
+
+<style lang="scss">
+.notification-enabling-pointer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  position: absolute;
+
+  top: 0;
+  left: 0;
+  z-index: 90;
+  width: 100%;
+  height: 100%;
+  background-color: #000000;
```

Comment:
Can we use variable here and below please? or at least shorten version (#000 instead of #000000).